### ISSUE TITLE
Drop down notifications

### ISSUE THREAD

**Issue by K4LCIFER** [thunder-app/thunder#1591]
### Is Your Feature Request Related to a Problem?

Currently, when notifications are received, I only get a vibration, and a pending notification symbol for the app is shown at the top, but I don't get a drop down that shows what the notification is — I have to open the notification drawer to be able to see it.

### Feature Description

I would like for there to be a drop down notification when a notification is received.

### Additional Context

Perhaps related to https://github.com/thunder-app/thunder/issues/219

**Comment by micahmo** [comment]
For devs: this is called a "head's up" notification (on Android). I spent just a little bit of timing looking into this with `flutter_local_notifications` and it wasn't immediately obvious how to do it. I tried changing both the `importance` and `priority`, but no luck so far. So this will require some more investigation by whomever eventually implements it.

**Comment by CTalvio** [comment]
Are we talking about "pop on screen"?

This setting can be enabled or disabled for each notification channel of an app on android. And whatever an app defaults to, can be overridden by the user.

You should be able to get the behaviour you want by going to settings>apps>thunder>notifications>channel(either messages or inbox), and enabling "pop on screen".

**Comment by micahmo** [comment]
Good tip, that worked for me!

That said, somehow some apps are able to set this by default. So there must be something in the Android API to configured the preferred preferences.

**Comment by CTalvio** [comment]
Yes, but it may be that after the channels are created, an update or the app cannot then change the settings.

I've seen apps that work around users turning off advertising notification channels, by creating entire new channels, infinitely, whenever the existing ones are set to mute or disable notifications the user does not want.

Obviously Thunder shouldn't do that, but it means you might need to uninstall the app entirely each time, to test "default" settings, as apps cannot change the settings except initially, for good reason.

**Comment by micahmo** [comment]
Great point!
